### ISSUE TITLE
Consistency seems to always match for rules that do not map to WCAG

### ISSUE THREAD

**Issue by Jym77** [act-rules/act-tools#38]
Looking at the implementation report for [Element with role attribute has required states and properties](https://www.w3.org/WAI/standards-guidelines/act/rules/4e8ab6/proposed/#implementations), several of us are listed as "Consistent" despite listing more SC than needed.

The rule now only maps to ARIA (used to also map to 1.3.1 and 4.1.2), but tools that still map to 1.3.1/4.1.2 are nonetheless reported as consistent. For example, [Alfa report](https://www.w3.org/WAI/standards-guidelines/act/implementations/alfa-assisted/#breakdown-4e8ab6) even state:
> It covers all 15 examples. The implementation correctly reports which success criteria are failed by this rule. See [understanding ACT consistency](https://www.w3.org/WAI/standards-guidelines/act/implementations/#understanding-act-consistency).
> 
> Success Criteria 
> | Expected | Reported |
> |---|---|
> | None | 1.3.1 Info and Relationships
> | | 4.1.2 Name, Role, Value
